### DISCUSSION TITLE
not able to select Qemu USB hard drive option

### DISCUSSION THREAD

**Discussion by Shahzaib2028** [utmapp/UTM#4239] [UTM Help (macOS)] [answered]
I am not able to select the Qemu USB hard drive. By pressing enter to select it seem to do nothing.

**Configuration**
* UTM Version: 
* OS Version: BigSur 11.3.1
* Intel or Apple Silicon? Apple M1

<img width="641" alt="Screenshot 2022-07-18 at 12 59 07 AM" src="https://user-images.githubusercontent.com/56338888/179422975-87446882-a077-4f6f-8f1d-f35c8bb431da.png">

These are the settings I use.
<img width="904" alt="Screenshot 2022-07-18 at 12 58 26 AM" src="https://user-images.githubusercontent.com/56338888/179422993-c8b180ae-861b-41dd-b36a-de1be095228a.png">

**Comment by osy** [comment, marked as answer]
You made an ARM virtual machine but tried to boot an AMD64 (x86_64) ISO.
